RobotNavigation/MazeView.cs

```diff
@@ -21,15 +23,15 @@ public void Display()
 
                 for (int x = 0; x < _maze.Width; x++)
                 {
-                    switch (_maze.GetCell(x, y))
+                    switch (_position.GetPosition(_maze).Cell)
```

Comment:
You're wanting to display the _maze here:

switch (_maze[x,y].Type)
{
...
}

Reply:
I get an error when I do this